src/Termi-Commands/Commands/sysfetch.hpp

```diff
@@ -7,7 +7,8 @@
 
 #pragma once
 
-#ifdef __linux__
+#ifdef _WIN32
+#elif __linux__ || __FreeBSD__ || __OpenBSD__ || __NetBSD__
```

Comment:
I know this is used to fix problem with `info` struct but things like number of packages are only working on (GNU/)Linux so we need further testing to check if this code will cause some kind of exception

Reply:
The code mean:
ifdef macro _WIN32
do nothing
else if defined linux, bsd, ...
include the libraries below